policies/allowed_vm_sku/policy.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "displayName": "HMCTS Allowed VM SKU sizes",
+    "policyType": "BuiltIn",
+    "mode": "Indexed",
+    "parameters": {
+      "allowedSkus": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Allowed SKU names",
+          "description": "HMCTS allowed VM SKU sizes",
```

Comment:
comma at the end of the `description` line, json not happy with it 😉

Reply:
comma has been moved